src/database/migrations/20190914173000-modify-request-table-with-new-columns.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.addColumn(
+      'Requests',
+      'passportNumber', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
```

Comment:
I feel anything that has do with a user's profile should only be on the user's table